index.html

```diff
@@ -204,44 +204,47 @@ <h2 id="media-text">
         </div>
         <div class="languages">
             <h3 id="lang">
-                <p>Languages</p>
-                <img src="images\Union.svg">
+                <p role="article">Languages</p>
+                <img src="images\Union.svg" alt="">

```

Comment:
:red_circle:  Please make sure you add an appropriate alternative text to every image alt attribute. That's useful for interacting or understanding. One way to help determine if an image should have a null alt is to ask yourself: If the image was removed, would the user still get all the information from the page? In this case, this arrow helps us to expand/collapse this section